modules/post/multi/gather/aws_keys.rb

```diff
@@ -24,7 +24,7 @@ def initialize(info = {})
         },
         'License' => MSF_LICENSE,
         'Author' => [ 'Jon Hart <jon_hart[at]rapid7.com>' ],
-        'SessionTypes' => %w[shell meterpreter],
+        'SessionTypes' => %w[shell linux meterpreter],
```

Comment:
Presumably this was meant to be added to the `Platform` info Hash key, not `SessionTypes`.